frontend/integration-tests/tests/devconsole/dev-perspective.scenario.ts

```diff
@@ -1,12 +1,12 @@
 import { browser, $ } from 'protractor';
 import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
-import {
-  switchPerspective,
-  Perspective,
-} from '../../views/devconsole-view/dev-perspective.view';
+import { switchPerspective, Perspective } from '../../views/devconsole-view/dev-perspective.view';
+
+export const pageSidebar = $('#page-sidebar .pf-c-nav .pf-c-nav__list');
+export const sideHeader = $('#page-sidebar .oc-nav-header h1');
 
 describe('Application Launcher Menu', () => {
-  beforeAll(async() => {
+  beforeAll(async () => {
```

Comment:
nit: no need to add space before function parentheses. Please remove it because of this frontend tests are failing on this PR.

@sanketpathak  run ```yarn lint --fix``` on this PR. It will fix all the lint error like this.